libs/ledger-live-common/src/device-core/managerApi/entities/IdEntity.ts

```diff
@@ -0,0 +1 @@
+export type IdEntity = number;
```

Comment:
[ASK] I'm not sure it's really an "entity", as it just represents the type of id we use in our Manager API domain, and not an object living (being created, updated, deleted) in the doman. We can ask @mbertin-ledger how we should name it ?

Reply:
makes sense, could just be `Id` then I guess